### ISSUE TITLE
Decision on possible UI/CSS framework

### ISSUE THREAD

**Issue by dorono** [ResistanceCalendar/resistance-calendar-frontend#9]
Very new territory for me as it relates to React, but an equivalent of a Bootstrap, Material-UI, or basically, anything that we could use for out-of-the-box easy-to-implement UI and interactions which we know have already been tested on all major browsers and devices. 

Here are some of the newer ones I found from a quick Google search:
https://hashnode.com/post/10-best-reactjs-ui-frameworks-for-rapid-prototyping-cit49tqx414z89c53equ4zc5k

Does anyone have any experience with any of these or any other UI/CSS frameworks?

**Comment by pfarnach** [comment]
I've used Material-UI a good amount and it's solid and straightforward but overwriting their styles can be a pain (lots of inline styles and `!important`s).  They allow for theming via a provider which is handy for global overwrites.  

I've been playing around with Semantic-UI on a side project and I've enjoyed it and seems like a solid choice as well.  

I'm not crazy about the Material UI design language in general so I'm partial to Semantic-UI (or others) but I'm open to whatever works best for everyone.
